v19.2/create-table.md

```diff
@@ -264,74 +258,108 @@ In this example, we use `ON DELETE CASCADE` (i.e., when row referenced by a fore
+        id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+        city STRING,
+        name STRING,
+        address STRING,
+        credit_card STRING,
+        dl STRING UNIQUE CHECK (LENGTH(dl) < 8)
+);
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> CREATE TABLE orders (
-    id INT PRIMARY KEY,
-    customer_id INT REFERENCES customers(id) ON DELETE CASCADE
-  );
+> CREATE TABLE vehicles (
+        id UUID NOT NULL DEFAULT gen_random_uuid(),
+        city STRING NOT NULL,
+        type STRING,
+        owner_id UUID REFERENCES users(id) ON DELETE CASCADE,
+        creation_time TIMESTAMP,
+        status STRING,
+        current_location STRING,
+        ext JSONB,
+        CONSTRAINT "primary" PRIMARY KEY (city ASC, id ASC),
+        INDEX vehicles_auto_index_fk_city_ref_users (city ASC, owner_id ASC),
+        INVERTED INDEX ix_vehicle_ext (ext),
+        FAMILY "primary" (id, city, type, owner_id, creation_time, status, current_location, ext)
+);
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW CREATE orders;
-~~~
-
-~~~
-+------------+--------------------------------------------------------------------------+
-| table_name |                             create_statement                             |
-+------------+--------------------------------------------------------------------------+
-| orders     | CREATE TABLE orders (                                                    |
-|            |                                                                          |
-|            |     id INT NOT NULL,                                                     |
-|            |                                                                          |
-|            |     customer_id INT NULL,                                                |
-|            |                                                                          |
-|            |     CONSTRAINT "primary" PRIMARY KEY (id ASC),                           |
-|            |                                                                          |
-|            |     CONSTRAINT fk_customer_id_ref_customers FOREIGN KEY (customer_id)    |
-|            | REFERENCES customers (id) ON DELETE CASCADE,                             |
-|            |                                                                          |
-|            |     INDEX orders_auto_index_fk_customer_id_ref_customers (customer_id    |
-|            | ASC),                                                                    |
-|            |                                                                          |
-|            |     FAMILY "primary" (id, customer_id)                                   |
-|            |                                                                          |
-|            | )                                                                        |
-+------------+--------------------------------------------------------------------------+
+> SHOW CREATE vehicles;
```

Comment:
This could be slightly more specific as `SHOW CREATE TABLE` but IDK if it matters.  Either way it works!

Reply:
Done.